### DISCUSSION TITLE
Command line option -search with re-search

### DISCUSSION THREAD

**Discussion by JoWeb-de** [sumatrapdfreader/sumatrapdf#3356] [Q&A] [answered]
I'm using the command line options **-search** to open a PDF file and position on the first found search argument (in this case: look for error number "0C7"). Example:
"c:\program files\sumatrapdf\sumatrapdf.exe -search 0C7 t:\booksneu\ieah700.pdf"

Two questions:
1. Can you please document this option in Sumatra documentation?
2. Can you please enable re-search via function key "F3" or by putting the search argument into the small search window so you don't have to type it again?

Please forget No. 2: I just found that pressing ctrl-F does just this!

Jo

**Comment by GitHubRulesOK** [comment]
Hmm 
Well spotted
your correct that the "Manual" page does not have an entry for https://www.sumatrapdfreader.org/docs/Version-history#:~:text=cursor%20over%20scrollbar-,add%C2%A0%2Dsearch%C2%A0,-cmd%2Dline%20option   
Often additions are not added / picked up until released so for example there is in pre-release `-disable-auto-rotation`

@kjk perhaps review which command line options are "non-experimental"

**Comment by kjk** [comment, marked as answer]
I've documented `-search` option.
